src/main/java/org/ibp/api/java/impl/middleware/derived/DerivedVariableServiceImpl.java

```diff
@@ -215,6 +217,9 @@ public DerivedVariableServiceImpl() {
 
 		}
 
+		// Update the dependent phenotypes as out of sync in batch
+		this.middlewareDatasetService.updateDependentPhenotypesAsOutOfSync(targetMeasurementVariable.getTermId(), observationUnitIds);
```

Comment:
Hi @mderamos modify execute() method's unit test to verify middlewareDatasetService.updateDependentPhenotypesAsOutOfSync is called.